File: Banco de Dados 2/Python/Atividade1/Indicador.py
class DadosTaxa:
    def __init__(self, idPais: str, ano: str, taxa: float):
        self.idPais = idPais
        self.ano = ano
        self.taxa = taxa
    
    def __str__(self):
        value = "Pais: " + self.idPais + " Ano: "+ self.ano +" Taxa: " + self.taxa
        return value

File: Banco de Dados 2/Java/atividade1/Indicador.java
package atividade1;
public class Indicador {
    private String pais;
    private Double taxa;
    
    public Indicador(){}
    
    public Indicador(String pais, Double taxa) {
        this.pais = pais;
        this.taxa = taxa;
    }

    public String getPais() {
        return pais;
    }

    public void setPais(String pais) {
        this.pais = pais;
    }

    public Double getTaxa() {
        return taxa;
    }

    public void setTaxa(Double taxa) {
        this.taxa = taxa;
    }

    
}
